README.md

```diff
@@ -560,8 +560,8 @@ This repository contains a curated list of awesome open source libraries that wi
 * [DataRobot](https://www.datarobot.com/) - Automated machine learning platform which enables users to build and deploy machine learning models.
 * [Datatron](https://datatron.com/) - Machine Learning Model Governance Platform for all your AI models in production for large Enterprises.
 * [Deep Cognition Deep Learning Studio](https://deepcognition.ai/) - E2E platform for deep learning.
-* [deepsense Safety](https://safety.deepsense.ai/) - AI-driven solution to increase worksite safety via safety procedure check, thread detection and hazardous zones monitoring.
-* [deepsense Quality](https://quality.deepsense.ai/) - Automating laborious quality control tasks.
+* [DeepMind](https://www.deepmind.com/) - DeepMind aims to solve intelligence, developing more general and capable problem-solving systems, known as artificial general intelligence (AGI).
```

Comment:
Deepmind is not a production mlops platform for productionising models so can remove

Reply:
> Deepmind is not a production mlops platform for productionising models so can remove

I am thinking about this issue as well. I recently checked their portfolio in terms of production tools such as https://github.com/deepmind/alphafold and https://github.com/deepmind/mujoco. Considering their impact in making pharmaceutical medicine, I think their products are already in the process of putting ML into real-world production and saving lives for patients. I suggest keeping it on the list @axsaucedo

Reply:
This list is for tooling to productionise ML (or privinding tooling for production ML) - there's many companies doing great things in ml but not mentioning in the list so can remove

Reply:
> This list is for tooling to productionise ML (or privinding tooling for production ML) - there's many companies doing great things in ml but not mentioning in the list so can remove

If Deepmind provides the tools to ML production such as https://github.com/deepmind/lab, can we still consider it? @axsaucedo

Reply:
Yes I see what you mean but from my knowledge deepmind's tools are more for experimentation and training - perhaps some of their RL tools could go in the RL section if they are open source, but I still don't see Deepmind as an MLOps vendor

Reply:
same as below, not a vendor, let's remove as we're going to go back and forth forever on these two

Reply:
In fact, Deepmind is a for-profit company... It indeed designs ML tools and helps with MLOps for research, business, and education. @axsaucedo I do not think there is enough reason to exclude it.

Reply:
My perspective is that it doesn't "provide ML services" in a similar way to the rest of the companies in the list, they are achieving a lot but still don't have an API or a platform tht companies can purchase and use. OpenAI is OK as they now have various AI API services, but deepmind doesn't - I see this not as a list of all companies doing great things in AI, but a list of MLOps vendors that offer services